packages/manager/modules/pci/src/projects/project/public-gateways/edit/edit.controller.js

```diff
@@ -10,6 +10,8 @@ export default class PciPublicGatewaysEditController {
     this.isEditing = false;
     this.ovhSubsidiary = this.coreConfig.getUser().ovhSubsidiary;
     this.selectedGatewaySize = null;
+    this.gatewayDetails = null;
+    this.getGatewayById();
```

Comment:
**nitpick:** `fetchGatewayDetails` or `setGatewayDetails` seems more appropriate. And moreover, the function doesn't accept any ID.